scripts/gulpfiles/build_tasks.js

```diff
@@ -691,14 +682,13 @@ function buildAdvancedCompilationTest() {
   }
 
   const srcs = [
-    TSC_OUTPUT_DIR + '/closure/goog/base_minimal.js',
-    TSC_OUTPUT_DIR + '/closure/goog/goog.js',
-    TSC_OUTPUT_DIR + '/core/**/*.js',
-    'blocks/**/*.js',
-    'generators/**/*.js',
+    TSC_OUTPUT_DIR + '/**/*.js',
     'tests/compile/main.js',
     'tests/compile/test_blocks.js',
   ];
+  const ignore = [
+    TSC_OUTPUT_DIR + '/closure/goog/base.js',  // Use base_minimal.js only.
+  ];
```

Comment:
Optional nit: Personally I think it would be clearer to provide the two files explicitly (goog & base_minimal) rather than excluding base.js

Reply:
That was more or less how it was before—with us having to explicitly enumerate each of the subdirectories of `build/` (now including `blocks/` and `generators/`.  I think it's a lot tidier to say "compile everything output by `tsc`, except for this one specific file."  (Especially when that one specific file will likely go away before long.)

Reply:
Ah ok, if it's theoretically going away soon that makes a lot of sense. SGTM!